dlc/src/main/scala/org/bitcoins/dlc/DLCTimeouts.scala

```diff
@@ -10,4 +11,8 @@ case class DLCTimeouts(
     penaltyTimeout: Int,
     contractMaturity: BlockStampWithFuture,
     contractTimeout: BlockStampWithFuture
-)
+) extends NetworkElement {
+  override def bytes: ByteVector = {
+    ByteVector(penaltyTimeout.byteValue) ++ contractMaturity.toUInt32.bytes ++ contractTimeout.toUInt32.bytes
```

Comment:
I believe you should be doing `UInt32(penaltyTimeout).bytes` on the first thing to get this correct